docs/5.0/kubernetes-access.md

```diff
@@ -5,61 +5,244 @@ description: How to set up and configure Teleport for Kubernetes access with SSO
-  for both SSH and Kubernetes API.
-* Users RBAC roles are always synchronized between SSH and Kubernetes, making
-  it easier to implement policies like _developers must not access production
-  data_.
-* Complete `kubectl` auditing and session recordings for `kubectl exec`
-* Multi Kubernetes Support. Login to Teleport once and quickly switch between multiple K8s clusters using [`tsh kube login`](cli-docs.md#tsh-kube-login)
+* Users can login once using [`tsh login`](cli-docs.md#tsh-login) and
+  switch between multiple clusters using [`tsh kube login`](cli-docs.md#tsh-kube-login).
+* Admins can use roles to implement policies like `developers must not access production`.
+* Teleport captures `kubectl` events and session recordings for `kubectl exec`.
 
-## Start Here
+## Deploy Teleport In Kubernetes
 
-Before we dive into setup, we've a few options to help guide you. We've created some example
-Teleport Kubernetes Access configurations for different scenarios, all the way from solo developers
-accessing minikube through to large enterprises accessing hundreds of Kubernetes clusters.
+![teleport-kubernetes-inside](img/teleport-k8s-pod.svg)
+
+=== "Community"
 
-Teleport Kubernetes service requires Auth & Proxy to be setup. This can be inside or
-outside of k8s.
+    Setup and Install Teleport Helm chart repository.
+    ```bash
+    helm repo add teleport https://charts.releases.teleport.dev
+    helm install teleport teleport/teleport
+    ```
+
+    Download Demo Helm Value and Update Teleport Pod.
+    ```bash
+    curl https://github.com/gravitational/teleport/blob/master/examples/chart/teleport/quickstart/values-community.yaml --output values.yaml
+    helm upgrade -f values.yaml teleport teleport/teleport
+    ```
 
-Example Kubernetes Cluster Configurations:
+    Check that the Teleport Service is running.
+    ```bash
+    kubectl get service
+    # NAME       TYPE       CLUSTER-IP    EXTERNAL-IP   PORT(S)                                                                      AGE
+    # teleport   NodePort   10.20.28.76   <none>        3025:32676/TCP,3026:30207/TCP,3023:32763/TCP,3024:31710/TCP,3080:32333/TCP   89m
+    ```
 
-* Connecting to a local MicroK8s Cluster:
-     * Use Option 2, once setup you'll need to connect to Ingress via TODO
+    !!! Tip
 
-* Connecting to one EKS/GKS hosted Kubernetes:
-      * Use Option 2, once setup you'll need to connect to Ingress via TODO
+        If Teleport isn't starting you can tail Kubernetes logs for more info `kubectl logs --follow deployments/teleport`
+        or `kubectl get pods`<br>_Example: `kubectl describe pod teleport-5f5f989b96-9khzq`_
 
-* Connecting to multiple Kubernetes clusters in one cloud/region:
-      * Use Option 1, once setup you'll need to connect via the root Teleport service.
+    Forward ports to provide UI and CLI access.
+    ```
+    # Change teleport-699d68645c-wpqm7 to the name of the Pod
+    kubectl port-forward teleport-699d68645c-wpqm7 6379:32334
```

Comment:
Why use port forward here? The easiest way to either use a load balancer `kubectl expose` on GKE or on bare metal use node port.